### PR TITLE
Add to `tctl workflow reset` reference

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: docs/tctl/workflow/reset.md [hunk 1]
```diff
@@ -44,8 +49,8 @@ tctl workflow reset --run_id <id>
 
 ### `--event_id`
 
-Specify the `eventId` of any event after `WorkflowTaskStarted` to which you want to reset.
-Valid values are `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.
+Specify the `eventId` number of the event to which you want to reset.
+The event type must be one of: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.
```
Comment (lorensr) [inline docs/tctl/workflow/reset.md:53]: Clarifying that `WorkflowTaskCompleted` is not a valid value for `--event_id`. Must be a number.

Comment (djmagee) [inline docs/tctl/workflow/reset.md:53]:
```suggestion
Specify the `eventId` of the event to which you want to reset.
The event type must be one of the following: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, or `WorkflowTaskTimeout`.
```

File: docs/tctl/workflow/reset.md [hunk 2]
```diff
@@ -16,7 +16,12 @@ To run multiple Reset operations at once, see [`tctl workflow reset-batch`](/tct
 
 `tctl workflow reset <modifiers>`
 
-The following modifiers control the behavior of the command.
+The modifiers control the behavior of the command.
```
Comment (djmagee) [inline docs/tctl/workflow/reset.md:19]:
```suggestion
```

**Comment by flossypurse** [comment]
Maybe just confirm - but it looks like this was picked up in the tctl "v2" update https://github.com/temporalio/documentation/pull/1555

Can see a preview of it here -> https://deploy-preview-1555--mystifying-fermi-1bc096.netlify.app/tctl/workflow#reset

close in favor of the larger update?

**Comment by DJSanti** [comment]
Yeah, I already have this.

**Pull request closed without merging**